hugo/static/css/styles.css

```diff
@@ -288,3 +288,17 @@ iframe#matoOpOut {
 @media (min-width: 492px)  { iframe#matoOpOut { height: 20vh; } }
 @media (min-width: 768px)  { iframe#matoOpOut { height: 15vh; } }
 @media (min-width: 1024px) { iframe#matoOpOut { height: 14vh; } }
+
+/* Styles for markdown formatting of inline code and code blocks */
+code {
+  color: black;  /* A shade of gray #6e6e6e;*/
+  background-color: var(--scilife-light-grey);
+}
+
+pre {
+  color: black !important;
+  background-color: var(--scilife-light-grey) !important; 
+  -moz-tab-size: 4;
+  -o-tab-size: 4;
+  tab-size: 4;
```

Comment:
Could add the following params for styling, feel free to choose none/some/edit of course:
```
  border: 5px solid var(--scilife-medium-grey);
  border-radius: 10px; 
  line-height: 1.5;
  padding: 5px;
```